diffpy/pdfmorph/morphs/morph.py

```diff
@@ -120,8 +122,8 @@ def __init__(self, config=None):
     def morph(self, xobj, yobj, xref, yref):
         '''Morph arrays objective or reference.
 
-        xobj, yobj  --  Objective arrays.
-        xref, yref  --  Reference arrays.
+        xobj, yobj  --  Objective arrays (input 1).
```

Comment:
keep this reference to which order the arrays are added, but again change the name